src/components/InsightCard/index.js

```diff
@@ -38,13 +39,13 @@ const InsightCard = ({
           {description}
         </Typography>
       )}
-      <Link className={classes.link} href={href}>
-        {linkdescription && (
+      {ctaText && href && (
+        <Link className={classes.link} href={href}>
           <Typography variant="subtitle2" className={classes.linkText}>
-            {linkdescription}
+            {ctaText}
           </Typography>
-        )}
-      </Link>
+        </Link>
```

Comment:
I think it should be the other way [`Link` inside `Typography`](https://material-ui.com/components/links/#simple-links),

Reply:
not related to this PR, but you're right